### ISSUE TITLE
Image inside card should have an alt attribute

### ISSUE THREAD

**Issue by yalagand** [adobe/spectrum-web-components#731]
Assuming the title attribute serves as an appropriate label, rendered img tag in a card should have empty alt='' to mark it as decorative. Failing which, it is considered non decorative and missing a label.
